content/en/docs/setup/production-environment/tools/kubeadm/dual-stack-support.md

```diff
@@ -118,7 +122,7 @@ nodeRegistration:
 `advertiseAddress` in JoinConfiguration.controlPlane specifies the IP address that the API Server will advertise it is listening on. The value of `advertiseAddress` equals the `--apiserver-advertise-address` flag of `kubeadm join`.
 
 ```shell
-kubeadm join --config=kubeadm-config.yaml ...
+kubeadm join --config=kubeadm-config.yaml
```

Comment:
Could we:
- omit all the other changes
- emphasize that readers might want to include a `--discovery-token` option here

?
(I haven't checked that you can combine `--discovery-token` and `--config`; I think you can though).

Reply:
What about the new changes?

> emphasize that readers might want to include a --discovery-token option here
I gave it a shot and got some errors.
```
$ kubeadm join --config=kubeadm-config.yaml --discovery-token-ca-cert-hash sha256:a4863cde706cfc580a439f842cc65d5ef112b7b2be31628513a9881cf0d9fe0e
can not mix '--config' with arguments [discovery-token-ca-cert-hash]
To see the stack trace of this error execute with --v=5 or higher
```

Reply:
I also think that separation of concern is not my cup of tea. I prefer all configuration in one file. :)

Reply:
Thanks for testing this.